dm/worker/relay.go

```diff
@@ -70,13 +103,20 @@ func NewRelayHolder(cfg *Config) *RelayHolder {
 }
 
 // Init initializes the holder
-func (h *RelayHolder) Init() error {
+func (h *RealRelayHolder) Init(interceptors []purger.PurgeInterceptor) (purger.Purger, error) {
```

Comment:
how about changing the name to `relayHolder` (lower case)?

Reply:
Did you want to be exposed before?

Reply:
expose the interface (`RelayHolder`), but not expose the implementation (`relayHolder`)